aviary/subsystems/propulsion/gearbox/model/gearbox_premission.py

```diff
@@ -0,0 +1,64 @@
+import openmdao.api as om
+import numpy as np
+
+from aviary.variable_info.variables import Aircraft, Dynamic
+from aviary.utils.aviary_values import AviaryValues
+
+
+class GearboxPreMission(om.Group):
+    """
+    Calculate gearbox mass for a single gearbox
+    """
+
+    def initialize(self):
+        self.options.declare(
+            "aviary_inputs", types=AviaryValues,
+            desc="collection of Aircraft/Mission specific options",
+            default=None,
+        )
+        self.name = 'gearbox_premission'
+
+    def setup(self):
+
+        self.add_subsystem('gearbox_PRM',
+                           om.ExecComp('RPM_out = RPM_in / gear_ratio',
+                                       RPM_out={'val': 0.0, 'units': 'rpm'},
+                                       gear_ratio={'val': 1.0, 'units': None},
+                                       RPM_in={'val': 0.0, 'units': 'rpm'}),
+                           promotes_inputs=[('RPM_in', Dynamic.Mission.RPM),
```

Comment:
How does a premission component have any Dynamic:* variables?

Reply:
`Dynamic.Mission.RPM` is a single value, chosen by the optimizer. `Motor_premission.py` has this same input. This works because we are assuming collective control for the props.

Reply:
I can change the motor and the gearbox input name to accept a `design.rpm` value, going to have to touch a bunch more files.